src/content/en/updates/2020/03/devtools.md

```diff
@@ -0,0 +1,83 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: TODO
+
+{# wf_updated_on: 2020-03-09 #}
+{# wf_published_on: 2020-03-09 #}
+{# wf_tags: chrome81, devtools, devtools-whatsnew #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: TODO #}
+{# wf_blink_components: Platform>DevTools #}
+
+# What's New In DevTools (Chrome 82) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+## Emulate vision deficiencies 
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/e847a020093fbd3c367eda02a7008feae5e28641 #}
+
+Send feedback to [Chromium issue #1003700](https://crbug.com/1003700).
```

Comment:
Note:

- This is currently a Google-only issue (not sure why though). We should either make it public or remove this line.
- Comment #19 in the issue has some TODOs for the documentation.

Reply:
Removed the line